webapp/src/routes/test/+page.svelte

```diff
@@ -1,18 +1,99 @@
 <script lang="ts">
-	interface Country {
-		name: string;
+	import ButtonBig from '$lib/components/ButtonBig.svelte';
+	import ButtonSmall from '$lib/components/ButtonSmall.svelte';
+
+	// Glob import all SVG files from the avatars directory
+	const avatars = import.meta.glob('$lib/assets/avatars/*.svg', { eager: true });
+
+	// Type definitions for the module imports
+	interface SvgModule {
+		default: string;
 	}
 
-	export let data: { countries: Country[]; error?: string };
+	// Extract the default export (the path) of each module
+	const avatarArray: string[] = Object.values(avatars).map((mod) => (mod as SvgModule).default);
 </script>
 
-{#if data.error}
-	<p>Error: {data.error}</p>
-{:else}
+<header class="text-center mb-8">
+	<h1 class="text-4xl font-bold font-contrail mb-4 text-black">Color Palette Showcase</h1>
+	<p class="text-lg text-black font-vampiro">
+		This page demonstrates the color palette defined in Tailwind CSS.
+	</p>
+</header>
+
+<main class="flex flex-col items-center gap-10">
+	<h1>Welcome to Flashbackfiesta!</h1>
+
 	<ul>
-		<h1>Countries</h1>
-		{#each data.countries as country}
-			<li>{country.name}</li>
+		{#each avatarArray as avatar}
+			<li><img src={avatar} alt="Avatar" /></li>
 		{/each}
 	</ul>
-{/if}
+
+	<ButtonSmall text="LET'S PLAY" accent_color="#54e5c4" />
```

Comment:
We should change the accent_color to red (#ff847c) because of the green background color